e2e-tests/README.md

```diff
@@ -0,0 +1,70 @@
+Normandy-e2e-tests
+====================
+
+
+Summary
+-------
+Suite of end-2-end tests for Mozilla's Normandy Control Shield UI. 
+
+
+Each script in the tests subdirectory exercises a different workflow of the UI. The tests are ordered 1-8. The tests run sequentially, and the first (test1) will run first, and the last (test8) test will run last.
+
+
+The workflows tested are:
+
+2. Filling out the recipe form
+3. Editing a recipe
+4. Approving a recipe
+5. Publishing a recipe
+6. Disabling a recipe
+7. Cloning a recipe
+8. Verifying that recipes created in the UI are created in the restful api endpoint
+9. Uploading a web extension
+
+Tests 7 and 8 are skipped, which is addressed in issues #11 and #9, respectively.
```

Comment:
These are Github issues on the old repo, right? These should be updated for the new repo.